### ISSUE TITLE
osc changes dont occur 

### ISSUE THREAD

**Issue by coffeeratmedia** [TheStaticTurtle/OSC2MEETER#1]
osc touch app is able to see changes made on from voice meeter but chanmges like mute/slider adjustments don't actually occon voice meeter and only  shows on the app i have attach a error that shows up when i have tryed to make change via the app
![Capture](https://user-images.githubusercontent.com/56743736/109076672-0ed69e00-774f-11eb-8bbe-8d127ac66b5a.PNG)

**Comment by TheStaticTurtle** [comment]
So voice meeter to touchosc works but not touchosc to voicemeeter ?
I've seen this particular exception before but not so many of them, coulvd you give more info on your system (ie 32bit/64bit, version of the sharposc dll that you downloaded, .....)
+What does the data you are sending looks like? Are you using my example?

**Comment by coffeeratmedia** [comment]
thats correct and yes i am using your example file cause i wanted to ensure things working before go mess with build my own osc file , as to what sharposc i am using the fiel u provided the .zip i am using 64bit wiindows 10 os this seem to be happen let me know if u need any more info

**Comment by TheStaticTurtle** [comment]
Honestly I'm a bit lost here the part that is causing the issue is the exact same thing as in the library example, I'll have to look at it more closely tomorrow
